src/providers/reversing-distances/reversing-distances.ts

```diff
@@ -0,0 +1,56 @@
+import { Injectable } from '@angular/core';
+import { TestCategory } from '@dvsa/mes-test-schema/categories/common/test-category';
+// TODO: Import CAT C types
+import { CatBEUniqueTypes } from '@dvsa/mes-test-schema/categories/BE';
+import { ReversingLengths } from './reversing-lengths.model';
+
+// TODO: refactor CAT BE to use this provider
+@Injectable()
+export class ReversingDistancesProvider {
+
+  public getDistanceLength(data: Object, category: TestCategory): ReversingLengths {
```

Comment:
use `object` type rather than `Object`

Reply:
FYI - https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html

Reply:
Good spot - thanks.